interface/patient_file/summary/record_disclosure.php

```diff
@@ -78,84 +78,86 @@ function submitform() {
-    <input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
+                    <input type='hidden' name='mode' value="disclosure" />
+
+                    <div class="form-group mt-3">
+                        <label><?php echo xlt('Date'); ?>:</label>
+                        <?php
+                        if ($editlid) {
+                            $dres = sqlQuery("select date,recipient,description,event from extended_log where id=?", array($editlid));
+                            $description = $dres["description"];
+                            $app_event = $dres["event"];
+                            $disc_date = $dres["date"];
+                            $recipient_name = $dres["recipient"];
+                            ?>
+                            <input type="hidden" name="disclosure_id" value="<?php echo attr($editlid); ?>">
+                            <input type="hidden" name="updatemode" value="disclosure_update">
+                            <input type='entry' size='20' class='datepicker form-control' name='dates' id='dates' value='<?php echo attr($disc_date);?>'/>&nbsp; <?php
+                        } else {
+                            ?> <input type='entry' size='20' class='datepicker form-control' name='dates' id='dates' value=''/>&nbsp;<?php
+                        } ?>
+                    </div>
+
+                    <div class="form-group mt-3">
+                        <label><?php echo xlt('Type of Disclosure'); ?>:</label>
+                        <?php
+                        if ($editlid) {
+                            //To incorporate the disclosure types  into the list_options listings
+                            generate_form_field(array('data_type' => 1,'field_id' => 'disclosure_type','list_id' => 'disclosure_type','fld_length' => '10','max_length' => '63','empty_title' => 'SKIP'), $app_event);
+                        } else {
+                            //To incorporate the disclosure types  into the list_options listings
+                            generate_form_field(array('data_type' => 1,'field_id' => 'disclosure_type','list_id' => 'disclosure_type','fld_length' => '10','max_length' => '63','empty_title' => 'SKIP'), $title);
+                        } ?>
+                    </div>
+
+                    <div class="form-group mt-3">
+                        <label><?php echo xlt('Recipient of the Disclosure'); ?>:</label>
+                        <?php
+                        if ($editlid) {
+                            ?> <input type="entry" class="form-control" name="recipient_name" size="20" value="<?php echo attr($recipient_name); ?>" />
+                            <?php
+                        } else {?>
+                            <input type="entry" class="form-control" name="recipient_name" size="20" value="" />
+                            <?php
```

Comment:
Hi @bradymiller  and @tywrenn 

I guess the `type="entry"` is invalid code syntax. What do I replace?

Reply:
There are so many of these in the codebase that this will need to be a separate PR.

Reply:
Did you mean to create a PR and focus on removing the `type="entry"` ?

Reply:
Yeah and replacing it with `type="text"`